Product/Production/Services/DocumentSubmissionCore/src/test/java/gov/hhs/fha/nhinc/docsubmission/outbound/deferred/request/StandardOutboundDocSubmissionDeferredRequestTest.java

```diff
@@ -155,129 +196,119 @@ public void testGetters() {
         RegistryResponseType regResponse = new RegistryResponseType();
-        regResponse.setStatus(NhincConstants.XDR_ACK_STATUS_MSG);
-
+        if (buildSuccessResponse) {
+            regResponse.setStatus(NhincConstants.XDR_ACK_STATUS_MSG);
+        } else {
+            regResponse.setStatus(NhincConstants.XDR_ACK_FAILURE_STATUS_MSG);
+        }
         XDRAcknowledgementType response = new XDRAcknowledgementType();
         response.setMessage(regResponse);
 
-        OutboundDocSubmissionDeferredRequestOrchestratable orchestratable = new OutboundDocSubmissionDeferredRequestOrchestratable(null);
+        OutboundDocSubmissionDeferredRequestOrchestratable orchestratable
+            = new OutboundDocSubmissionDeferredRequestOrchestratable(mockDelegate);
         orchestratable.setResponse(response);
 
         return orchestratable;
     }
 
-    private StandardOutboundDocSubmissionDeferredRequest createEntityDocSubmissionDeferredRequestOrchImpl() {
+    private StandardOutboundDocSubmissionDeferredRequest createDocSubmissionDeferredRequestOrchImpl() {
         return new StandardOutboundDocSubmissionDeferredRequest() {
-            protected XDRAuditLogger getXDRAuditLogger() {
-                return mockXDRLog;
+            @Override
+            protected DocSubmissionDeferredRequestAuditLogger getAuditLogger() {
+                return mockAuditLogger;
             }
 
+            @Override
             protected XDRPolicyChecker getXDRPolicyChecker() {
-                return mockPolicyCheck;
+                return mockPolicyChecker;
             }
 
+            @Override
             protected SubjectHelper getSubjectHelper() {
                 return mockSubjectHelper;
             }
 
+            @Override
             protected OutboundDocSubmissionDeferredRequestDelegate getOutboundDocSubmissionDeferredRequestDelegate() {
                 return mockDelegate;
             }
         };
     }
 
-    @Test
-    public void hasOutboundProcessingEvent() throws Exception {
-        Class<StandardOutboundDocSubmissionDeferredRequest> clazz = StandardOutboundDocSubmissionDeferredRequest.class;
-        Method method = clazz.getMethod("provideAndRegisterDocumentSetBAsyncRequest",
-                ProvideAndRegisterDocumentSetRequestType.class, AssertionType.class,  NhinTargetCommunitiesType.class,
-                UrlInfoType.class);
-        OutboundProcessingEvent annotation = method.getAnnotation(OutboundProcessingEvent.class);
-        assertNotNull(annotation);
-        assertEquals(DocSubmissionBaseEventDescriptionBuilder.class, annotation.beforeBuilder());
-        assertEquals(DocSubmissionArgTransformerBuilder.class, annotation.afterReturningBuilder());
-        assertEquals("Document Submission Deferred Request", annotation.serviceType());
-        assertEquals("", annotation.version());
+    private AssertionType createAssertion(String hcid) {
```

Comment:
This isn't meant for this PR, but just something to think about for future refactoring: all of the functions below look super familiar, I'm pretty sure we have them duplicated somewhere else?  We should be thinking about whether these test helper methods can be stored somewhere better.